src/components/atoms/Timeline/index.tsx

```diff
@@ -18,6 +17,7 @@ import {
   STRONG_SCALE_WIDTH,
 } from "./constants";
 import { useTimelineInteraction } from "./hooks";
+import { ScaleList } from "./ScaleList";
```

Comment:
So you're of the school that doesn't use export default, it's nice to have the IDE complement working for you.